polyfill/lib/ecmascript.mjs

```diff
@@ -2264,12 +2082,12 @@ export function TemporalDateTimeToString(isoDateTime, calendar, precision, showC
 }
 
 export function TemporalMonthDayToString(monthDay, showCalendar = 'auto') {
-  const month = ISODateTimePartString(GetSlot(monthDay, ISO_MONTH));
-  const day = ISODateTimePartString(GetSlot(monthDay, ISO_DAY));
+  const month = ISODateTimePartString(GetSlot(monthDay, ISO_DATE).month);
```

Comment:
I couldn't add a suggestion here, but seems like a good spot to use the same `*String` pattern used above.